README.md

```diff
@@ -62,7 +62,10 @@ $ operator-sdk new app-operator --api-version=app.example.com/v1alpha1 --kind=Ap
 $ cd app-operator
 
 # Build and push the app-operator image to a public registry such as quay.io
+# We use sed to replace the placeholder image name that is generated the creation point of the operator
+# The sed operation only needs to be down once during the first build
 $ operator-sdk build quay.io/example/app-operator
+$ sed -i 's/REPLACE_IMAGE/quay.io\/example\/app-operator/g' deploy/operator.yaml
```

Comment:
Can you use a different separator so we don't have to escape `/` which makes the image name confusing e.g
`sed -i 's|REPLACE_IMAGE|quay.io/example/app-operator|g'`